cohere_v3_vector/index-vectors-only-mapping.json

```diff
@@ -0,0 +1,29 @@
+{
+  "settings": {
+    {# non-serverless-index-settings-marker-start #}{%- if build_flavor != "serverless" or serverless_operator == true -%}
+      {% if preload_pagecache %}
+    "index.store.preload": [ "vec", "vex", "vem"],
+      {% endif %}
+    "index.number_of_shards": {{number_of_shards | default(1)}},
+    "index.number_of_replicas": {{number_of_replicas | default(0)}}
+    {%- endif -%}{# non-serverless-index-settings-marker-end #}
+  },
+  "mappings": {
+    "dynamic": false,
+    "_source": {
+      "enabled": false
+    },
+    "properties": {
+      "emb": {
+        "type": "dense_vector",
+        "element_type": "float",
+        "dims": 1024,
+        "index": true,
+        "similarity": "max_inner_product",
+        "index_options": {
+          "type": {{ vector_index_type | default("int8_hnsw") | tojson }}
```

Comment:
The embeddings are normalized already so we can use dot_product directly

Reply:
https://github.com/elastic/rally-tracks/pull/575#discussion_r1517068264

Reply:
oh that's weird, we can use `cosine` for now then but I wonder now if the published dataset has the right embeddings. Maybe they just disabled it but that's not possible to do using the API and the v3 models.